lang/main-fr.json

```diff
@@ -65,26 +65,26 @@
         "dontShowAgain": ""
     },
     "connectingOverlay": {
-        "joiningRoom": "Connexion à la réunion..."
+        "joiningRoom": "Connexion à la réunion…"
     },
     "connection": {
         "ATTACHED": "Attachée",
         "AUTHENTICATING": "Authentification en cours",
         "AUTHFAIL": "Échec de l'authentification",
-        "CONNECTED": "Connecté",
+        "CONNECTED": "Connecté·e",
```

Comment:
Same here? Why is this change?

Reply:
It is gender-related, some people in France try to gender-neutralize the French language and that's a way of doing it. No way of doing it make perfect consensus in French writing community.

Reply:
Other forms of gender-inclusive language sometimes appear in the current translation, but not everywhere. For example: "Oups! vous avez été expulsé(e)"
I think it is important to decide on a rule and to apply it consistently, and I suggest choosing the midpoint for this.